lib/appmap/config.rb

```diff
@@ -324,15 +335,22 @@ def load(config_data)
 
         if config_data['functions']
           config_params[:functions] = config_data['functions'].map do |function_data|
-            package = function_data['package']
-            cls = function_data['class']
-            functions = function_data['function'] || function_data['functions']
-            raise %q(AppMap config 'function' element should specify 'package', 'class' and 'function' or 'functions') unless package && cls && functions
+            function_name = function_data['name']
+            package, cls, functions = []
+            if function_name
+              package, cls, _, function = Util.parse_function_name(function_name)
+              functions = Array(function)
+            else
+              package = function_data['package']
+              cls = function_data['class']
+              functions = function_data['function'] || function_data['functions']
+              raise %q(AppMap config 'function' element should specify 'package', 'class' and 'function' or 'functions') unless package && cls && functions
+            end
 
             functions = Array(functions).map(&:to_sym)
             labels = function_data['label'] || function_data['labels']
             labels = Array(labels).map(&:to_s) if labels
-            Function.new(package, cls, labels, functions)
+            Function.new(package, cls, labels, functions, function_data['builtin'], function_data['require'])
```

Comment:
How did you figure out you needed to add `require`? Does it deserve a test case, too?

Reply:
I too wonder why and when would it be needed.